packages/leancode_lint/README.md

```diff
@@ -0,0 +1,44 @@
+# leancode_lint
+
+[![leancode_lint pub.dev badge][pub-badge]][pub-badge-link]
+
+Lint rules used internally in LeanCode projects.
+
+## Usage
+
+Add `leancode_lint` as normal dependency to access `unawaited` function.
+
+```yaml
+dependencies:
+  leancode_lint: ^1.0.0
+```
+
+### Apps
+
+Include rules in `analysis_options.yaml` of your project.
+If necessary you can exclude some files from analysis.
+
+```dart
+include: package:leancode_lint/analysis_options.yaml
+
+# Optional
+analyzer:
+  exclude:
+    - "**/*.g.dart"
+```
+
+### Packages
+
+For packages include `analysis_options_package.yaml` instead.
```

Comment:
```suggestion
For packages include the `analysis_options_package.yaml`.
```

_instead_ refers to something from the previous same-level section. You can't expect the reader to read all sections, only those that they're interested in.